plugins/Dashboard/javascripts/widgetMenu.js

```diff
@@ -25,7 +25,7 @@ widgetsHelper.getAvailableWidgets = function () {
                 widgetsHelper.availableWidgets = data;
             }
         );
-        ajaxRequest.send(true);
+        ajaxRequest.send();
```

Comment:
If tests work right now, it is more like by accident as the data could be loaded before already but this is not always the case